### PR TITLE
get customers with filters

### PR DESCRIPTION
**Pull request by kachmashk**

### CODE REVIEW

File: api/db/repositories/customerRepository.js [hunk 1]
```diff
@@ -1,25 +1,52 @@
-const Customers = require('@db/dbHelper').Customers;
+const { Customers } = require('@db/dbHelper')
+const { Sequelize } = require('@db/dbHelper')
 
-exports.Customers = Customers;
+const Op = Sequelize.Op
+exports.Customers = Customers
 
 exports.createCustomer = async (customerData) => {
```
Comment (bionanek) [inline api/db/repositories/customerRepository.js]: remove async

Comment (bionanek) [reply api/db/repositories/customerRepository.js]: in the whole file where there are no awaits going on

File: api/domain/customersDomain.js [hunk 2]
```diff
@@ -1,52 +1,56 @@
-const customerRepo = require('@repos/customerRepository');
-const { Verification, Resources } = require('@verify/verification');
-const { IdNotFound, SequelizeError } = require('@helpers/errors');
+const customerRepo = require('@repos/customerRepository')
+const { Verification, Resources } = require('@verify/verification')
+const { IdNotFound, SequelizeError } = require('@helpers/errors')
 
 exports.create = async (customerData) => {
```
Comment (bionanek) [inline api/domain/customersDomain.js]: asyncs

**Review by bionanek — CHANGES_REQUESTED** [review]

File: api/domain/customersDomain.js (hunk 2 above)
Comment (kachmashk) [reply api/domain/customersDomain.js]: done

File: api/db/repositories/customerRepository.js (hunk 1 above)
Comment (kachmashk) [reply api/db/repositories/customerRepository.js]: done

**Review by bionanek — APPROVED** [review]

**Pull request merged**